### PR TITLE
Change wapi to sapi

### PR DESCRIPTION
**Pull request by r1se**
Change wapi to sapi according:
https://www.binance.com/en/support/announcement/f45dde7da58b473aa885349946bed269

### CODE REVIEW

**Comment by RoanBrand** [comment]
+1, only few days remaining

File: v2/withdraw_service.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func (s *CreateWithdrawService) Name(v string) *CreateWithdrawService {
 func (s *CreateWithdrawService) Do(ctx context.Context) (*CreateWithdrawResponse, error) {
 	r := &request{
 		method:   "POST",
-		endpoint: "/wapi/v3/withdraw.html",
+		endpoint: "/sapi/v1/capital/withdraw/apply",
 		secType:  secTypeSigned,
 	}
 	r.setParam("asset", s.asset)
```
Comment (RoanBrand) [inline v2/withdraw_service.go:78]: https://binance-docs.github.io/apidocs/spot/en/#withdraw-sapi

Seems `asset` changed to `coin`

**Comment by anugrah-shukla** [comment]
Only few days left, @adshao any updates on this?

**Comment by adshao** [comment]
> Only few days left, @adshao any updates on this?

I will do it if no one implement this PR by the end of this week.

**Comment by adshao** [comment]
> Only few days left, @adshao any updates on this?

Implemented in #282

**Pull request closed without merging**